internal/run/run.go

```diff
@@ -516,6 +516,9 @@ func overlayDocument(ctx context.Context, schema string, overlayFiles []string,
 	currentBase := schema
 	for _, overlayFile := range overlayFiles {
 		applyPath := getTempApplyPath(overlayFile)
+		if err := os.MkdirAll(filepath.Dir(applyPath), os.ModePerm); err != nil {
```

Comment:
Actually I think you can just check that the temp dir is made going into overlays. Shouldn't have to do it for each path

Reply:
oh ya that's better